### ISSUE TITLE
containertest.sh fails several ways - invalid nginx configuration

### ISSUE THREAD

**Issue by rfay** [drud/docker.nginx-php-fpm-local#37]
## What happened (or feature request):

The phpmyadmin nightly was broken on ddev today. On further investigation it turned out that phpmyadmin was broken because it attempts to listen on port 1081. However, an earlier test (for docker.nginx-php-fpm-local had already grabbed port 1081, and this was never cleaned up, or the container stopped.

Further investigation of the problem with docker.nginx-php-fpm-local shows that:

* test/containertest.sh does not clean up the container it starts as a daemon in "testing use of custom nginx config"
* In that last test /etc/nginx/sites-enabled/nginx-site.conf contains this invalid content, which prevents nginx from starting: `docroot is /var/www/html/potato in custom conf`
* Probably the whole DOCROOT= approach is not working?

## Related source links or issues:

**Comment by rfay** [comment]
This problem was introduced in https://github.com/drud/docker.nginx-php-fpm-local/pull/35 yesterday. 

The containertest.sh (nicely) moved a bunch of stuff out of the Makefile, but missed one line, the one that shuts down the container.

The invalid test-custom.conf (containing "docroot is /var/www/html/potato in custom conf ") was introduced in 7d5d06f1f416270c05bb0d818932a33d84324bb7 - it seems like a very odd thing to do, to use an invalid test configuration that crashes nginx, and then just check to see that it's there. But there's nothing recently new about this.

Bottom line is: We can probably reintroduce a single line back into container-test.sh and we'll be running again.

**Comment by rfay** [comment]
I pulled #38, but think we should at least consider the consequences or intent of the invalid site config in test.

**Comment by rfay** [comment]
Closing this as not terribly useful and lots of water under the bridge.
